src/app/enums/virtualization.enum.ts

```diff
@@ -23,6 +39,7 @@ export const virtualizationStatusLabels = new Map<VirtualizationStatus, string>(
 ]);
 
 export enum VirtualizationRemote {
+  Vm = 'VM',
```

Comment:
No available yet but expected to be supported by `virt.instance.image_choices` endpoint

Reply:
I think remote is a different thing. For dialog, pass in instance type and then filter images by `instance_types` on frontend.